gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title:  "Ideas for IMS - ERAS"
+show_main: false
+ideas_team: IMS
+---
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+*Suggested Mentors:* [Antonio del Mastro](https://bitbucket.org/aldebran/) , [Olena Persianova](https://bitbucket.org/OPersian/)
+
+*Difficulty:* Intermediate to Hard
+
+*Astronomy knowledge needed:* None beforehand, the student will be required to research relevant publications.
+
+*Programming skills:* advanced Python; basic Theano or TensorFlow; basic Django or Flask; experience with some ANN library, such as Keras, theanets or Lasagne.
+
+#### Description:
+
+Solar storms are responsible for disruption of satellite communication, and damage to space electronical equipments. The storms have to be taken into account also for EVA and habitat maintenance activities, as the higher levels of radiation brought by them have a detrimental effect on the crew member's health.
+
+Prediction of these storms are essential to prevent said damage. A lot of astronomical data is generated on a daily basis, and this could be used in conjunction with machine learning methods to predict solar storms.
+
+In this project, the student will be required to:
+
+* Using a machine learning approach, predict the *duration* and *intensity* solar storms:
+  * The student should use preferably an artificial neural networks approach (although alternatives, such as random forests, SVM, bayesian models or HMMs, can be considered).
+  * The predictions should be given with 1-3 weeks in advance of a storm (depending on viability).
```

Comment:
Nowadays, predict a storm one week in advance is really hard, three weeks is just luck.  I can believe that for high speed solar wind streams which follows more or less a 27 days pattern because the solar rotation, also a prediction of sunspots number could be done (not always right). But I doubt we could do so for a solar storm (understanding so as a flare, Coronal Mass Ejection or both...) with the current understanding of the physics and probability. On the day... sure, you could get a probability of an active region to flare or a filament to lift off... (eg. what [SolarMonitor produces everyday](http://solarmonitor.org/forecast.php?date=20160309)) - but going more than 48 hours is not here yet (maybe alphago could try it :) )